src/loader.rs

```diff
@@ -83,28 +115,41 @@ impl<L: ModuleLoader> ModuleStream<L> {
   fn append_module(&mut self, url: Url) {
     if !self.started.contains(&url) {
       self.started.insert(url.clone());
-      let url_ = url.clone();
-      let fut =
-        Box::pin(self.loader.load(url).and_then(|module_source| async move {
-          let module_info = match module_source {
-            ModuleLoad::Redirect(url) => ModuleInfo::Redirect(url),
-            ModuleLoad::Source {
-              source,
-              content_type,
-            } => {
-              let (deps, transpiled) =
-                get_deps_and_transpile(&url_, &source, &content_type)?;
-              ModuleInfo::Source(ModuleSource {
+      if url.scheme() == "data" {
```

Comment:
Instead of `if` and `if else`, just turn this into a `match url.scheme() {`

Reply:
Thanks :)